spec/views/my/_list_works.html.erb_spec.rb

```diff
@@ -4,7 +4,7 @@
   let(:work_title) { 'Work Title' }
   let(:coll_title) { 'Collection Title' }
   let(:collection) { FactoryGirl.build(:collection, id: '3197z497t', title: coll_title) }
-  let!(:work) { FactoryGirl.build(:work, id: '3197z511f', title: [work_title]) }
+  let!(:work) { FactoryGirl.create(:generic_work, id: '3197z511f', title: [work_title]) }
```

Comment:
@jcoyne , when I use .build with persisted?: true, the travis build still failed.. can we see the history of the PR?